core/templates/components/interaction-display/interaction-display.component.ts

```diff
@@ -57,6 +57,7 @@ export class InteractionDisplayComponent {
 
   ngAfterViewInit(): void {
     this.buildInteraction();
+    this.changeDetectorRef.detectChanges();
```

Comment:
Can you explain (for posterity) why this change was needed?

Reply:
I had already justified this [here](https://github.com/oppia/oppia/issues/13699#issuecomment-2031771112), but now that I've looked back at the code, I think it's not necessary since the changes are already being detected within the buildInteraction(). I'll remove it.